AUTHORS.md

```diff
@@ -241,6 +241,11 @@ Robotic Transformations (Dissamble) sound by
 Mekaal - https://opengameart.org/content/robotic-transformations
 Licensed under CC BY 3.0 http://creativecommons.org/licenses/by/3.0/
 
+SpaceEngine_Start_00 as colonized by
+Little Robot Sound Factory
+www.littlerobotsoundfactory.com
+Licensed under CC BY 3.0 http://creativecommons.org/licenses/by/3.0/
+
```

Comment:
This same information should also in org.openRealmOfStars.game.States.CreditsView.java in public static final String MAIN_CREDITS. Authors.md can be generated running OROS with --credits